flaml/searcher/blendsearch.py

```diff
@@ -279,28 +279,29 @@ def suggest(self, trial_id: str) -> Optional[Dict]:
                 # logger.debug(f"random config {config}")
                 skip = self._should_skip(choice, trial_id, config)

```

Comment:
why don't we break until we find one that we don't need to skip? It may get stuck in the for loop?

Reply:
yes, to avoid getting stuck.